### PR TITLE
Cropping

### PR DESCRIPTION
**Pull request by siirias**
nemo_loader: adds function to crop empty columns
utility: tool to calculate cell volumes

### CODE REVIEW

**Comment by tkarna** [comment]
looks good, test PEP8 style with flake8 ;)

File: siri_omen/nemo_reader.py [hunk 1]
```diff
@@ -383,7 +383,29 @@ def extract(self, var, lon, lat, z=0.0,
         # make sure we comply with the required metadata policy
         utility.assert_cube_metadata(output)
         return output
-
+def remove_null_indices(in_cube):
+    """
+    Searches dimenions which have only nulls, and trims the cube by removing those
```
Comment (tkarna) [inline siri_omen/nemo_reader.py]: typo

File: siri_omen/nemo_reader.py [hunk 2]
```diff
@@ -383,7 +383,29 @@ def extract(self, var, lon, lat, z=0.0,
...
+      all_but_this=tuple(filter(lambda x: x!=i,dimensions))
+      masks=[':']*len(dimension_lens)
+      axis_mask=~numpy.max(cube.data,axis=all_but_this).mask
+      masks[i]='axis_mask'
+      cut_command='cube=cube['
+      for c in masks:
+        cut_command+=c+','
+      cut_command=cut_command[:-1]+']'  # :-1 here to remove tha last comma from loop.
+      ldict={'cube':cube,'axis_mask':axis_mask}
+      exec(cut_command,globals(),ldict)   #which should cut the cube on current dimension,
```
Comment (tkarna) [inline siri_omen/nemo_reader.py]: I'm not a huge fan of python exec but I don't really mind

File: siri_omen/utility.py [hunk 3]
```diff
@@ -393,3 +393,21 @@ def compute_cube_statistics(reference, predicted):
     r = reference.data
     p = predicted_alinged.data
     return statistics.compute_statistics(r, p)
+
+
+def cube_volumes(cube):
+  """
+  caclulates volumes for each cell based on
```
Comment (tkarna) [inline siri_omen/utility.py]: typo

File: siri_omen/nemo_reader.py (hunk 1 above)
Comment (siirias) [reply siri_omen/nemo_reader.py]: fixed typo

File: siri_omen/nemo_reader.py (hunk 2 above)
Comment (siirias) [reply siri_omen/nemo_reader.py]: Replaced the exec solution with slices. I think it's lot cleaner now.

File: siri_omen/utility.py (hunk 3 above)
Comment (siirias) [reply siri_omen/utility.py]: fixed typo

**Comment by siirias** [comment]
> looks good, test PEP8 style with flake8 ;)

Yeh, cleaned up the code and tried to comfort to PEP-8 :)

**Comment by tkarna** [comment]
All good, merging.

**Pull request merged**